frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListsActionsBar/CodeListsActionsBar.tsx

```diff
@@ -40,6 +43,11 @@ export function CodeListsActionsBar({
         className={classes.searchField}
         size='sm'
         placeholder={t('app_content_library.code_lists.search_placeholder')}
+        onChange={(event: ChangeEvent<HTMLInputElement>) =>
+          onSetCodeListSearchPattern(event.target.value)
+        }
+        clearButtonLabel={t('app_content_library.code_lists.clear_search_button_label')}
+        onClear={() => onSetCodeListSearchPattern('')}
```

Comment:
Trenger vi denne?
```suggestion
```

Reply:
Ser sånn ut. Eller det spørs hvordan du ellers tenker vi skal nullstille søk-verdien? 😊

Reply:
Ah, den har en egen knapp for å nullstille? Jeg tenkte dette bare var en funksjon som ble kalt når brukeren fjernet innholdet. Men da er det jo riktig.